testsuites/syncgateway/functional/tests/conftest.py

```diff
@@ -535,10 +535,10 @@ def params_from_base_suite_setup(request):
     # clean up firewall rules if any ports blocked for server ssl testing
     clear_firewall_rules(cluster_config)
     # Stop all sync_gateway and sg_accels as test finished
-    c = cluster.Cluster(cluster_config)
+    #c = cluster.Cluster(cluster_config)
 
     # Comment this out to use skip-provisions
-    c.stop_sg_and_accel()
+    #c.stop_sg_and_accel()
```

Comment:
I think making these changes to your local is good to run --skip-provisioning.
You might want to reconsider merging it to the master.

Reply:
Sorry, that shouldn't be there, thanks for spotting it!